build-support/scripts/protobuf.sh

```diff
@@ -104,79 +78,40 @@ function main {
     return 0
 }
 
-# Installs the version of protoc specified by the first argument.
-#
-# Will set 'protoc_bin'
-function protoc_install {
```

Comment:
Goodbye terrible function. You had a short, useful life.

Reply:
Yeah, I realize this is coming on the heels of you fixing up the situation. I kind of wish I had known you were dealing with that hell and maybe I could have recommended going the `buf` route instead.